framework/validators/FileValidator.php

```
@@ -182,6 +193,9 @@ public function init()
         if ($this->wrongExtension === null) {
             $this->wrongExtension = Yii::t('yii', 'Only files with these extensions are allowed: {extensions}.');
         }
+        if ($this->mimeTypeMismatch === null) {
+            $this->mimeTypeMismatch = Yii::t('yii', '{file} appears to be corrupted. Please check if the file extension is correct.');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Yii' in method 'init'. (undefined)

(StaticAccess)